ui/Screen/Project/Source/CheckoutMergeRequestButton.svelte

```diff
@@ -49,18 +44,16 @@
     <p style="margin-bottom: 0.5rem;">
       To check out this merge request locally, run this in your terminal:
     </p>
-    <Hoverable bind:hovering={hover}>
-      <Copyable bind:this={copyable} showIcon={hover}>
-        <p
-          class="typo-text-small-mono"
-          style="color: var(--color-foreground-level-6); overflow: scroll">
-          {instructions}
-        </p>
-      </Copyable>
-    </Hoverable>
+    <Copyable bind:this={copyable} showIcon={false}>
+      <p
+        class="typo-text-small-mono"
+        style="color: var(--color-foreground-level-6); overflow-x: scroll; padding: .5rem .5rem .5rem .25rem">
```

Comment:
Does this need a ending semicolon?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1683/commits/7ab714b2ad576552d8a6fcf77ee9f33f5b5641a3